packages/agoric-cli/src/commands/vaults.js

```diff
@@ -65,16 +59,13 @@ export const makeVaultsCommand = async logger => {
     .requiredOption('--giveCollateral [number]', 'Collateral to give', Number)
     .requiredOption('--wantMinted [number]', 'Minted wants', Number)
     .option('--offerId [number]', 'Offer id', Number, Date.now())
+    .option('--collateralBrand [string]', 'Collateral brand key', 'IbcATOM')
     .action(async function (opts) {
       logger.warn('running with options', opts);
 
-      const spendAction = makeOpenSpendAction(
-        // @ts-expect-error xxx RpcRemote
-        agoricNames.brand,
-        opts,
-      );
+      const offer = Offers.vaults.OpenVault(agoricNames.brand, opts);
```

Comment:
`OpenVault` expects `opts.collateralBrandKey` but when I try with the vaults smoke test, we seem to still supply `opts.collateralBrand`.